script/testing/util/common.py

```diff
@@ -22,55 +26,83 @@ def run_command(command,
                          cwd=cwd)
 
     while p.poll() is None:
-        if stdout == subprocess.PIPE:
-            out = p.stdout.readline()
-            if out:
-                LOG.info(out.decode("utf-8").rstrip("\n"))
+        if printable:
+            if stdout == subprocess.PIPE:
+                out = p.stdout.readline()
+                if out:
+                    LOG.info(out.decode("utf-8").rstrip("\n"))
 
     rc = p.poll()
     return rc, p.stdout, p.stderr
 
 
-def check_port(port):
+def print_output(filename):
+    """ Print out contents of a file """
+    with open(filename) as file:
+        lines = file.readlines()
+        for line in lines:
+            LOG.info(line.strip())
+
+
+def format_time(timestamp):
+    return datetime.datetime.fromtimestamp(timestamp).strftime(
+        "%Y-%m-%d %H:%M:%S")
+
+
+def kill_pids_on_port(port):
+    for proc in psutil.process_iter():
```

Comment:
Add a warning here that you can't call this function unless you are root.

You could add a check:

```python
if os.getuid() != 0:
   raise Exception("Cannot call this function unless running as root!")
```

Reply:
added the following snippet to **all** the functions uses `psutil`
```py
if os.getuid() != 0:
   raise Exception("Cannot call this function unless running as root!")
```